realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -2351,42 +2351,31 @@ private OsResults lazyFindAll() {
      *
      * // Sort, Distinct, Limit
      * query.rawPredicate("name = 'Jane' SORT(lastName) DISTINCT(city) LIMIT(5)");
+     *
+     * // Arguments
+     * query.rawPredicate("name = $0 AND age > $1", "Jane", 18);
```

Comment:
I would also add a line to the actual doc about this instead of hiding it in the examples.